src/App.tsx

```diff
@@ -15,48 +16,114 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+enum SortType {
+  NONE,
+  ALPABET,
+  LENGTH,
+}
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  let visibleGoods = [...goods];
+
+  switch (sortType) {
+    case SortType.ALPABET:
+      visibleGoods.sort((good1, good2) => (good1.localeCompare(good2)));
+      break;
+
+    case SortType.LENGTH:
+      visibleGoods.sort((good1, good2) => (good1.length - good2.length));
+      break;
+
+    case SortType.NONE:
+      visibleGoods = [...goods];
+      break;
+
+    default:
+      break;
+  }
+
+  if (isReversed) {
+    visibleGoods.reverse();
+  }
+
+  // eslint-disable-next-line no-console
+  console.log(sortType, isReversed);
+
+  return visibleGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortType, setSortType] = useState(SortType.NONE);
+  const [isReversed, setIsReversed] = useState(false);
+
   return (
     <div className="section content">
       <div className="buttons">
         <button
           type="button"
-          className="button is-info is-light"
+          // eslint-disable-next-line max-len
+          className={classNames('button is-info', { 'is-light': sortType !== SortType.ALPABET })}
+          onClick={() => {
+            setSortType(SortType.ALPABET);
+          }}
```

Comment:
```suggestion
          onClick={() => setSortType(SortType.ALPABET)}
```
you can leave it on the same line

Reply:
can't do it because of linter